### ISSUE TITLE
Problem after trying to upgrade copyq on Ubuntu 22.04: requires libqt5waylandclient5 >= 5.14.4 but have 5.15.3-1 and can't get higher

### ISSUE THREAD

**Issue by cpsyctc2** [hluk/CopyQ#2752]
**Describe the bug**
Copyq was working fine (one of my favourite and most used apps) but I saw that the latest release had improved things so I pulled copyq_8.0.0_Debian_12-1_amd64.deb onto my Ubuntu 22.04.4 LTS machine that I keep pretty well updated.  Now, after running as root:  
`dpkg -i copyq_8.0.0_Debian_12-1_amd64.deb`
I have this:
```
```The following packages have unmet dependencies.
 copyq : Depends: libqt5waylandclient5 (>= 5.15.4) but 5.15.3-1 is to be installed
E: Unmet dependencies. Try 'apt --fix-broken install' with no packages (or specify a solution).
```

I have tried that but the only fix it offers is to remove CopyQ

**To Reproduce**
As above.

**Expected behavior**
I naively expected a seamless upgrade of CopyQ which has always been the case in the past

**Version, OS and Environment**
Sadly, I no longer have a working CopyQ but I assume I was on the one before 8.0 (though I'm not clear why the sourceforge download was giving me 8.0 not 9.0)
Ubuntu 22.04.4 LTS

**Additional context**
I can't think of anything.  I do my updates/upgrades using topgrade pretty well daily and had done one earlier today but that only upgraded some R packages so I guess any change that affected wayland happened earlier.

TIA!

Chris

**Comment by hluk** [comment]
Please install the package from official repository for your distribution or PPA. See: https://copyq.readthedocs.io/en/latest/installation.html

**Comment by cpsyctc2** [comment]
Thanks.  I wanted the latest version so I used the instructions:
```
sudo apt install software-properties-common python-software-properties
sudo add-apt-repository ppa:hluk/copyq
sudo apt update
sudo apt install copyq
```
Which did the trick. 

I don't know about other Ubuntu releases but on mine, i.e. 22.04.4 LTS, python-software-properties is no longer available and has been replaced by software-properties-common.  It turned that I already had the latest version of that package so that caused no issues for me.

 Many thanks both for this point and also because CopyQ is brilliant and saves me hours every week!  Chris
